app/main/index.ts

```diff
@@ -1,21 +1,25 @@
 'use strict';
-const path = require('path');
-const fs = require('fs');
-
-const electron = require('electron');
-const windowStateKeeper = require('electron-window-state');
-const isDev = require('electron-is-dev');
-const appMenu = require('./menu');
-const { appUpdater } = require('./autoupdater');
+import { sentryInit } from '../renderer/js/utils/sentry-util';
+import { appUpdater } from './autoupdater';
+import { setAutoLaunch } from './startup';
+
+import windowStateKeeper = require('electron-window-state');
```

Comment:
I think I have read it somewhere that it's good practice to put the app dependencies first (path, fs, other modules).

Reply:
Yes, it is generally better to organize the imports.

Reply:
We had a linting rule which is why this was changed.